packages/core/ios/Tests/DdSdkTests.swift

```diff
@@ -383,12 +383,15 @@ internal class DdSdkTests: XCTestCase {
     }
 
     func testBuildFirstPartyHosts() {
-        let configuration: DdSdkConfiguration = .mockAny(additionalConfig: ["_dd.first_party_hosts": ["example.com", "datadog.com"]])
+        let configuration: DdSdkConfiguration = .mockAny(additionalConfig: ["_dd.first_party_hosts": [
+            ["match": "example.com", "propagatorTypes": ["datadog"]],
+            ["match": "datadog.com",  "propagatorTypes": ["b3multi", "tracecontext"]]
```

Comment:
probably worth putting all possible propagatorTypes in the test

Reply:
I will also add more tests with 
- badly formatted propagatorType
- duplicate match key with different propagatorTypes